storage/src/tests/distributor/distributortest.cpp

```diff
@@ -639,28 +639,31 @@ Distributor_Test::mergeStatsAreAccumulatedDuringDatabaseIteration()
         NodeMaintenanceStats wanted;
         wanted.syncing = 1;
         wanted.copyingOut = 2;
+        wanted.total = 3;
         CPPUNIT_ASSERT_EQUAL(wanted, stats.perNodeStats.forNode(0, makeBucketSpace()));
     }
     {
         NodeMaintenanceStats wanted;
         wanted.movingOut = 1;
+        wanted.total = 1;
         CPPUNIT_ASSERT_EQUAL(wanted, stats.perNodeStats.forNode(1, makeBucketSpace()));
     }
     {
         NodeMaintenanceStats wanted;
         wanted.syncing = 1;
         wanted.copyingIn = 2;
+        wanted.total = 1;
```

Comment:
Side note: we have to ensure that our cluster controller logic handles the (perhaps paradoxical) case where a distributor reports more pending buckets than it has in total. This being caused by `copyingIn`.